src/databricks/labs/ucx/mounts/list_mounts.py

```diff
@@ -0,0 +1,31 @@
+import logging
```

Comment:
move this file to `hive_metastore` package. this is the closest package it can belong to.

Reply:
What about creating a dedicated package for it? Either placing it in a `filesystem` package like dbutils.fs or more simply `mounts`